sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerWriteSchemaTransformProvider.java

```diff
@@ -147,7 +180,7 @@ public PCollectionRowTuple expand(@NonNull PCollectionRowTuple input) {
   @Override
   public @UnknownKeyFor @NonNull @Initialized List<@UnknownKeyFor @NonNull @Initialized String>
       outputCollectionNames() {
-    return Collections.singletonList("failures");
+    return Arrays.asList("failures", "errors");
```

Comment:
we should use tupletags, not raw strings in general

Reply:
Hey @johnjcasey, I was looking into this. It looks like PCollectionRowTuple only supports String tags, the method does not have support for TupleTags. That might be the reason these were implemented with Strings in the first place.

https://github.com/apache/beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/values/PCollectionRowTuple.java